tests-with-coverage/pom.xml

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.acme</groupId>
+    <artifactId>tests-with-coverage</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus.version>0.16.0</quarkus.version>
```

Comment:
`0.18.0` please

Reply:
Oops. Thought about it and forgot. Corrected

Reply:
Hmmm, shouldn't it be `999-SNAPSHOT` even? it's targeting the `development` branch.

Reply:
@fabricepipart Quarkus version needs to be 999-SNAPSHOT as @gsmet pointed out

Reply:
Changed